### PR TITLE
switch uses of grid & column classes to digitalmarketplace-govuk-frontend equivalents

### PR DESCRIPTION
**Pull request by risicle**
https://trello.com/c/ek1zy7Ce

Needed a couple of extended fixes to some table-based pages, helped by some advice from @lfdebrux . Other than that, I looked at all changed pages and they seemed to be visually identical.

### CODE REVIEW

**Comment by risicle** [comment]
Oh forgot about the tests - will get back on this soon...

File: app/templates/buyers/brief_overview.html [hunk 1]
```diff
@@ -15,10 +15,10 @@
 {% endblock %}
 
 {% block mainContent %}
-  <div class="grid-row">
+  <div class="govuk-grid-row">
     {% block before_heading %}
       {% if delete_requested %}
-        <div class="column-one-whole">
+        <div class="govuk-grid-column-one-whole">
```
Comment (lfdebrux) [inline app/templates/buyers/brief_overview.html]: Looking at the [table of grid widths in govuk-frontend](https://github.com/alphagov/govuk-frontend/blob/v2.13.0/src/settings/_measurements.scss#L23-L30) I don't think `govuk-grid-column-one-whole` is included in govuk-frontend

**Comment by lfdebrux** [comment]
We should now be able to remove some or all of the following Sass:

- [`_grid_layout.scss`](https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/1dfba29af508b41d1ad4fe02d2104b7b01441e6e/app/assets/scss/application.scss#L6)
- [`toolkit/_grids.scss`](https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/1dfba29af508b41d1ad4fe02d2104b7b01441e6e/app/assets/scss/application.scss#L27)
- [`overrides/_page-headings.scss`](https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/1dfba29af508b41d1ad4fe02d2104b7b01441e6e/app/assets/scss/application.scss#L65)

Have you tried this?

File: app/templates/buyers/brief_overview.html (hunk 1 above)
Comment (risicle) [reply app/templates/buyers/brief_overview.html]: Ah interesting so it's actually doing nothing, eh?

Comment (lfdebrux) [reply app/templates/buyers/brief_overview.html]: Yeah I think so

Comment (lfdebrux) [reply app/templates/buyers/brief_overview.html]: Should be replaced with `govuk-grid-column-full`

Comment (risicle) [reply app/templates/buyers/brief_overview.html]: :+1: thanks

**Comment by risicle** [comment]
> Have you tried this?

I have now. It appears to work ok, but I haven't gone through it as thoroughly as I did switching over the pages in the first place, because that takes a while...

**Comment by lfdebrux** [comment]
> > Have you tried this?
> 
> I have now. It appears to work ok, but I haven't gone through it as thoroughly as I did switching over the pages in the first place, because that takes a while...

If you push up the change as a separate commit I'll check them over

**Comment by risicle** [comment]
Done.

**Comment by lfdebrux** [comment]
Hmm okay I think we should keep `overrides/_page-headings` in as it adds spacing between the page heading and the breadcrumbs that is currently missing, but will added once we replace govuk_template with govuk-frontend's template.

Otherwise these changes look good to me!

**Pull request merged**